tests/phpunit/CRM/Core/ResourcesTest.php

```diff
@@ -167,8 +148,7 @@ public function testAddScriptURL(): void {
       ->addScriptUrl('/whiz/foo%20bar.js', 0, 'testAddScriptURL')
       ->addScriptUrl('/whizbang/foo%20bar.js', 0, 'testAddScriptURL');
 
-    $smarty = CRM_Core_Smarty::singleton();
-    $actual = $smarty->fetch('string:{crmRegion name=testAddScriptURL}{/crmRegion}');
+    $actual = CRM_Utils_String::parseOneOffStringThroughSmarty('{crmRegion name=testAddScriptURL}{/crmRegion}');
```

Comment:
Doesn't this need quotes for name key?

Reply:
well spotted @seamuslee001 - there were still a bunch of those in the file that hadn't bubbled up to being 'the thing currently causing the problem' - but I pushed up a fix